agreement/selector.go

```diff
@@ -64,19 +64,19 @@ func membership(l LedgerReader, addr basics.Address, r basics.Round, p period, s
 
 	record, err := l.BalanceRecord(balanceRound, addr)
 	if err != nil {
-		err = fmt.Errorf("Service.initializeVote (r=%v): Failed to obtain balance record for address %v in round %v: %v", r, addr, balanceRound, err)
+		err = fmt.Errorf("Service.initializeVote (r=%v): Failed to obtain balance record for address %v in round %d: %v", r, addr, balanceRound, err)
```

Comment:
The first %v here (and below) should be changed too, right?

Reply:
right.